vocode/streaming/audio/audio_service.py

```diff
@@ -3,21 +3,22 @@
 
 from vocode.streaming.models.audio import AudioServiceConfig
 from vocode.streaming.audio.base_audio_service import BaseThreadAsyncAudioService
+from vocode.streaming.models.audio_encoding import AudioEncoding
 
 
 class AudioService(BaseThreadAsyncAudioService[AudioServiceConfig]):
     """Audio Service"""
 
     def __init__(
         self,
-        conversation_id: str,
         audio_service_config: AudioServiceConfig,
         logger: Optional[logging.Logger] = None,
     ):
-        super().__init__(conversation_id, audio_service_config, logger)
+        super().__init__(audio_service_config, logger)
 
     def process(self, chunk: bytes) -> bytes:
         """No processing"""
+        self.audio_bytes += chunk
```

Comment:
This is getting microphone audio only (we need add speaker output as well but need more work we can do as needed)

Reply:
we need do it is proper way (issue here is we use audio_service to capture audio not handling sending audio to speaker. This need another refactor make audio service fontend for both audio in and out otherwise synchronization is hard.

Reply:
will add as backlog task.

Reply:
https://tenyx.atlassian.net/browse/TEN-222